bundle-workflow/src/test_workflow/perf_test_suite.py

```diff
@@ -0,0 +1,30 @@
+import os
+import subprocess
+
+
+class PerformanceTestSuite:
```

Comment:
class doesn't match filename

Reply:
Renamed the class name to `PerfTestSuite`